app/src/main/java/com/example/notes2text/myapplication/User.java

```diff
@@ -0,0 +1,13 @@
+package com.example.notes2text.myapplication;
+
+public class User {
+    protected String username;
+    protected String password;
```

Comment:
For more security on password, consider making password private, and have a protected function to check user input against stored password.